upgrade_versions.py

```diff
@@ -241,6 +273,10 @@ def main(args):
             LOG.exception("Failed to update versions")
             return e.errno
 
+    release_date = datetime.today().date().isoformat()
+    push_new_versions(versions_repo, release_date, push_repo_url,
+                      push_repo_branch, committer_name, committer_email)
```

Comment:
this will be pushed to a versions fork into our bot's account? if so, let's keep it then since no harm is made unless we submit it to a PR